cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,25 @@
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+// CONTRACT_ALIAS_MAP is a map from a contract alias to a contract address
+pub const CONTRACT_ALIAS_MAP: Map<&str, String> = Map::new("contract_alias_map");
+
+// CHAIN_TO_CHAIN_CHANNEL_MAP is a map from a source<>destination chain pair to its respective channel id
+pub const CHAIN_TO_CHAIN_CHANNEL_MAP: Map<(&str, &str), String> =
+    Map::new("chain_to_chain_channel_map");
+
+// CHANNEL_TO_CHAIN_CHAIN_MAP is a map from a channel id on a source chain to its respective destination chain
+pub const CHANNEL_TO_CHAIN_CHAIN_MAP: Map<(&str, &str), String> =
+    Map::new("channel_to_chain_chain_map");
+
+// NATIVE_DENOM_TO_IBC_DENOM_MAP is a map from a native cosmos chain denom to its respective osmosis ibc denom
+pub const NATIVE_DENOM_TO_IBC_DENOM_MAP: Map<&str, String> = Map::new("osmosis_denom_map");
```

Comment:
assuming a direct ibc transfer, right? This is the one that can be calculated via:
```rust 
let chain_channel = CHANNEL_TO_CHAIN_CHAIN_MAP.load((chain_name, "osmosis"))?;
sha256sum(format!("transfer/{chain_channel}/{denom}"))
```
right?

Reply:
We had previously wrote up in the design doc that this was a required registry, but I believe you are right (if you change load in your example above to take both the chain channel and the chain name)